doc/read-the-docs-site/tutorials/english-auction-example.rst

```diff
@@ -0,0 +1,332 @@
+.. _english_auction_tutorial:
+
+Plutus Script for an Auction Smart Contract
+==========================================================
+
+Overview
+------------
+
+In this page we look at the Plutus Tx code for a smart contract that controls the auction of an asset, which can be executed on the Cardano blockchain.
+
+Plutus Tx is a high level language for writing the validation logic of the contract, the logic that determines whether a transaction is allowed to spend a UTXO.
+Plutus Tx is not a new language, but rather a subset of Haskell, and it is compiled into Plutus Core, a low level language based on higher-order polymorphic lambda calculus.
+Plutus Core is the code that runs on-chain, i.e., by every node validating the transaction, using an interpreter known as the CEK machine.
+A Plutus Core program included in a Cardano transaction is often referred to as Plutus script or Plutus validator.
+
+To develop and deploy a smart contract, you'd also need off-chain code for building and submitting transactions, querying for available UTXOs and so on.
+You may also want a frontend for your smart contract for better user experiences.
+The frontend and backend of the off-chain part of smart contracts are not covered in this page.
+
+Before we get to the Plutus Tx code, let's briefly go over some basic concepts, including UTXO, datum, redeemer and script context.
+
+The EUTXO Model, Datum, Redeemer and Script Context
+--------------------------------
+
+UTXO (unspent transaction output) is the ledger model used by some blockchains, including bitcoin.
+A UTXO is produced by a transaction, is immutable, and can only be spent once by another transaction.
+In the classic UTXO model, a UTXO contains a wallet address and a value (e.g., some amount of Ada and/or other tokens), and is identified by the wallet address.
+It can be spent by a transaction, if the transaction is signed by the private key of the wallet address.
+
+The EUTXO (Extended UTXO) model introduces a new kind of UTXO: script UTXO.
+A script UTXO contains a value, a script (usually a Plutus script) and a piece of data called *datum*, and is identified by the hash of the script.
+For a transaction to spend it, the transaction must provide a piece of input data to the script, referred to as the *redeemer*.
+The script is then run, and it must succeed in order for the transaction to be allowed to spend the UTXO.
+Besides the redeemer, the script also has access to the datum contained in the UTXO, as well as the details of the transaction trying to spend it (referred to as *script context*).
+
+Note that the only thing a Plutus script does is to determine whether a transaction can spend the script UTXO that contains the script. It is *not* responsible for such things as deciding whether it can spend a different UTXO, checking that the input value in a transaction equals the output value, or updating the state of the smart contract.
+Consider it a pure function that returns ``Bool``.
```

Comment:
what does `checking that the input value in a transaction equals the output value` exactly mean? 

I thought it was possible that a script can do that for the current transaction, but *cannot* do that for any other transaction?

Reply:
I'll need to confirm the answer to this question @bezirg . Maybe I'll ask the question on plutus-core-dev channel.